scripts/remove-gotags-comments/main.go

```diff
@@ -39,6 +39,7 @@ func run(swaggerPath string) error {
 			//   - When gotags appears on its own.
 			//   - When gotags appears at the end of another comment (preceded by \n\n).
 			for _, prefix := range []string{"\\n\\n", ""} {
+				swaggerBytes = bytes.ReplaceAll(swaggerBytes, []byte(fmt.Sprintf("%s@gotags: %sclass:\\\"%s\\\" eventstream:\\\"observation\\\"%s", prefix, wrapper, classification, wrapper)), nil)
```

Comment:
Maybe add a comment inside the loop to say that we need to cover both the case where the class is set on its own, and where it's set with the eventstream?